uaclient/status.py

```diff
@@ -678,5 +676,18 @@ def format_json_status(status: "Dict[str, Any]") -> str:
 
     # We don't need the origin info in the json output
     status.pop("origin", "")
+    return status
+
+
+def format_json_status(status: "Dict[str, Any]") -> str:
+    from uaclient.util import DatetimeAwareJSONEncoder
+
+    return json.dumps(
+        _format_status_output(status), cls=DatetimeAwareJSONEncoder
+    )
+
+
+def format_yaml_status(status: "Dict[str, Any]") -> str:
+    import yaml
 
-    return json.dumps(status, cls=DatetimeAwareJSONEncoder)
+    return yaml.dump(_format_status_output(status))
```

Comment:
It seems that yaml is already parsing dates in an appropriate format. However, if you think we should make the data parsing works exactly as the json definition, let me know and I can code a loader for it